### DISCUSSION TITLE
Paradigma pellematic

### DISCUSSION THREAD

**Discussion by alele-de** [dominikamann/oekofen-pellematic-compact#38] [Ideas]
Hi Dominik,

I have an Paradigma Peleo Optima, it´s identical in construction to the ökofen with, may be, the exception of the software. Modbus is active, there is no separate JSON activation.
If I start the interface with the correct IP-Adesss I get the logon screen. Unfortunately I have no user/password, I guess it´s the default one, which I don´t no.
Do you think it is possible to try the oekofen integration or have you heard about the ident paradigma heater?
Or what can I do to check the modbus functionality?

Thank you and kind regards,
Alex

**Comment by dominikamann** [comment]
Hi Alex,

if there is no JSON interface, I don't see how my integration could work. Perhaps you can ask the manufacturer directly whether this option is available with an update. Alternatively, you can of course access it directly via Modbus. However, I have no experience here. For examples, see https://www.home-assistant.io/integrations/modbus/. FYI: In the Ökofen instruction paper of my heater, it is pointed out that the use of Modbus (not JSON) has a negative effect on the life time of the display ... . Let me know if I can help you further.

Dominik

**Reply by lucaferrucci** [reply]
hello, good morning, the paradigma has the same software as all the okofen without any difference, can you send a photo of the boiler display screen?

**Comment by alele-de** [comment]
Hi Luca, interesting to hear. Enclosed you will find a screen to the software data. KR, Alex

![Peleo_Optima_Paradigma](https://github.com/dominikamann/oekofen-pellematic-compact/assets/5948961/3ad6826e-76f5-4612-aeda-1fbacc3bfb93)

**Comment by lucaferrucci** [comment]
ok, the problem is that you have a very, very old software version, I advise you to update, the new version has completely redesigned both the interface and the way the machine works

**Comment by alele-de** [comment]
ok, thanks for the hint. My heating engineer won´t update it at the moment, why ever. I contact paradigma directly and hope for a quick response. If the machine works "better" than i think it´s mandatory to force an update.

**Comment by lucaferrucci** [comment]
if I'm not mistaken your machine should be identical to an okofen pellematic compact, I'm almost sure that you can download the okofen software and run the update, I can provide you with the update file, but it can only be done locally with a USB stick, if If you feel up to it, I can provide you with everything you need
